jsfiles/card.js

```diff
@@ -0,0 +1,143 @@
+const projects = {
+  first: {
+    name: 'First Project',
+    desc: `first project description <br> Lorem Ipsum is simply. 
+           Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, 
+           when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.`,
```

Comment:
- [Optional] Please add the missing popup as the previous reviewer requested, so as to make your portfolio looks professional and presentable. Happy coding :+1: